docs/dashboards/custom-chart.md

```diff
@@ -0,0 +1,164 @@
+# Custom Chart
+
+The Custom Chartm Block is an advanced Block that allows you to create custom data visualization by utilizing [Vega-Lite](https://vega.github.io/vega-lite/). If you would like a specific visualization that's not already a Losant Dashboard Block, the Custom Chart Block gives you the ultimate support for a wide array of visualization types.
+
+![Data Table Scatter Plot](../images/workflows/custom-chart-data-table-scatter-plot.png "Data Table Scatter Plot")
+
+## Configuration
+
+Configuring the Custom Chart Block is broken up into three sections ...
+
+### Choosing an Application
+
+First, choose an application that owns the devices and/or the data tables you would like to query.
+
+![Select Application](/images/workflows/custom-chart-select-application.png "Select Application")
+
+### Datasets
+
+![Datasets](/images/workflows/custom-chart-datasets.png "Datasets")
+
+#### Time Series Datasets
+
+Time series datasets provide the value of a single attribute from a selected device over a duration of time at a selected resolution. The values returned from the query will be returned as an array of objects in the form of `{ time: <time>, value: <value> }`. The value returned will reflect the selected attribute's value specified at a specific time.
+
```

Comment:
Might be helpful to add an example of what the dataset would look like. 

Like you could add this:

```

[
      { "time": Thurs Feb 18 2016 18:00:00 GMT-0500 (EST), "value": 48.8 },
      { "time": Thurs Feb 18 2016 19:00:00 GMT-0500 (EST), "value": 49.7 },
      { "time": Thurs Feb 18 2016 20:00:00 GMT-0500 (EST), "value": 51.0 },
      { "time": Thurs Feb 18 2016 21:00:00 GMT-0500 (EST), "value": 51.7 },
      { "time": Thurs Feb 18 2016 22:00:00 GMT-0500 (EST), "value": 50.2 },
      ...
  ]
```